data/colorfulclouds

```diff
@@ -0,0 +1,2 @@
+caiyunai.com
```

Comment:
tracker detected: `s22.cnzz.com/z_stat.php`.
we may need to create a new file for cnzz trackers.

Reply:
traffic analyser detected: `http://oct.pocoiq.cn/com.snowplowanalytics.snowplow/tp2`
need to create a new file for this domain.